### DISCUSSION TITLE
Dependency dashboard: add links to their changelogs

### DISCUSSION THREAD

**Discussion by ikatyang** [renovatebot/renovate#22143] [Suggest an Idea]
**What would you like Renovate to be able to do?**
Add links to their changelogs in the master issue. Currently, it's hard to understand what did they change from their title.

**Describe the solution you'd like**
N/A

**Describe alternatives you've considered**
N/A

**Additional context**
Something like the following:

## Pending Approval

These PRs will be created by Renovate only once you click their checkbox below.

 - [ ] chore(deps): update dependency @types/node to v10 (--> [changelog](#) <--)

**Comment by rarkins** [comment]
Main challenge here: PRs can have multiple updates, so there's not "one" link

Second challenge: The changelogs you see in PRs are heavily massaged/filtered and we can't change to those
